include/zephyr/drivers/lora.h

```diff
@@ -46,13 +46,36 @@ enum lora_coding_rate {
 };
 
 struct lora_modem_config {
+	/* Frequency in Hz to use for transceiving */
 	uint32_t frequency;
+
+	/* The bandwidth to use for transceiving */
 	enum lora_signal_bandwidth bandwidth;
+
+	/* The data-rate to use for transceiving */
 	enum lora_datarate datarate;
+
+	/* The coding rate to use for transceiving */
 	enum lora_coding_rate coding_rate;
+
+	/* Length of the preamble */
 	uint16_t preamble_len;
+
+	/* TX-power in dBm  to use for transmission */
 	int8_t tx_power;
+
+	/* Set to true for transmission, false for receiving */
 	bool tx;
+
+	/* Invert the IQ. Usually can be set to false, but should be set to true for sending a downlink. */
```

Comment:
What situation is this comment referring to? LoRa doesn't have a "downlink" concept.
It's fine to mention that this is true for LoRaWAN's PHY layer on downlinks only, but it shouldn't be implied that this is somehow a convention for LoRa itself.